examples/yaml-highlight/yaml_hl.py

```diff
@@ -37,14 +37,14 @@ def __setstate__(self, state):
 class YAMLHighlight:
 
     def __init__(self, options):
-        config = yaml.load(file(options.config, 'rb').read())
+        config = yaml.load(open(options.config, 'rb').read())
         self.style = config[options.style]
         if options.input:
-            self.input = file(options.input, 'rb')
+            self.input = open(options.input, 'rb')
```

Comment:
Is this one and the next one closed somewhere?

Reply:
(at least in `__del__` or so)

Reply:
Of course, it's much better and safer to turn this example into a context manager handling this properly on exit...

Reply:
Or accept an open file-like object as an option and handle open/close via CM on that level.

Reply:
Does this change resolve it?